store/sqlstore/channel_store.go

```diff
@@ -1665,46 +1660,14 @@ func (s SqlChannelStore) GetPinnedPostCount(channelId string, allowFromCache boo
 }
 
 func (s SqlChannelStore) InvalidateGuestCount(channelId string) {
-	channelGuestCountsCache.Remove(channelId)
-	if s.metrics != nil {
-		s.metrics.IncrementMemCacheInvalidationCounter("Channel Guest Counts - Remove by ChannelId")
-	}
 }
 
 func (s SqlChannelStore) GetGuestCountFromCache(channelId string) int64 {
```

Comment:
And remove it here too.

Reply:
I think we need it because its in the ChannelStore inferface.  should i delete the function in the interface?

Reply:
Yes, you should remove from both places.

Reply:
done!